war/src/main/java/io/lumeer/engine/controller/CollectionMetadataFacade.java

```diff
@@ -570,9 +570,17 @@ private Object checkAndConvertAttributeValue(final Attribute attribute, final Ob
          return afterCheck;
       }
 
+      // no encoding required
+      if (value == null) {
+         return null;
+      }
+
+      // other types
       final List<String> constraintConfigurations = attribute.getConstraints();
 
       final ConstraintManager constraintManager = new ConstraintManager(constraintConfigurations);
+      constraintManager.setLocale(Locale.getDefault());
```

Comment:
`ConstraintManager` almost never uses the locale, and objects using it might have more information about user Locale.